jina/serve/runtimes/gateway/http/app.py

```diff
@@ -15,6 +17,7 @@
 
 
 def get_fastapi_app(
+    args: 'argparse.Namespace',
```

Comment:
why is this needed ? if any specific parameter is really needed, it should be added to this func signature and to gateway constructor